.github/workflows/test.yml

```diff
@@ -0,0 +1,80 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+
+      - name: Install dependencies
+        run: npm install --location=global bslint
+
+      - name: Lint
+        run: npm run lint
+
+  test-coverage:
+    name: Test Coverage
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+
+      - name: Install dependencies
+        run: npm install --location=global nyc codecov
+
+      - name: Rebuild
+        run: npm rebuild
+
+      - name: Test
+        run: npm run test-ci
+
+#      - name: Coverage
+#        uses: coverallsapp/github-action@master
+#        with:
+#          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  build-test:
+    name: Build & Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node: [14.x, 16.x, 18.x]
```

Comment:
might be time already to bump these up: https://github.com/nodejs/release#release-schedule

Reply:
Good thought.